src/main/java/com/puppycrawl/tools/checkstyle/ConfigurationLoader.java

```diff
@@ -495,5 +496,18 @@ public void endElement(String uri,
                 }
             }
         }
+
+        /**
+         * Util method to recheck attribute in module.
+         * @param module modeul to check
+         * @param attributeName name of atteribute in module to find
+         * @return true if attribute is present in module
+         */
+        private boolean containsAttribute(Configuration module, String attributeName) {
```

Comment:
To me, I think it makes more sense for this to be a new method in `Configuration`.